src/components/hero/hero.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Hero } from "./hero";
+
+const title = "Ce pasi ai de urmat";
+const subtitle = "Pentru a te putea ajuta ...";
+const icon = <img src={require("./covid-all.svg")} alt={"foo"} />;
```

Comment:
We used this way of importing svgs in this repo: https://github.com/code4romania/taskforce-fe-components/blob/master/src/components/header/header.js#L4  
Please change the import and also add a relevant alt attribute :)

Reply:
yep, realised this while implementing the header component  👍